rdmo/core/serializers.py

```diff
@@ -38,7 +38,7 @@ def to_representation(self, instance):
         response = super().to_representation(instance)
 
         for markdown_field in self.markdown_fields:
-            if markdown_field in response and response[markdown_field]:
+            if response.get(markdown_field):
```

Comment:
I wasn't sure about these auto-changes by ruff.
Is this really equivalent?

What did `if markdown_field in response and response[markdown_field]` really test?

- there needs to be a key called `markdown_field` in response
- the value of `markdown_field` needs to be truthy

`response.get(markdown_field)` returns the value of markdown_field or None, the if clause tests if it is truthy.

So this should be safe, what do you think?

Reply:
yes, there were several of these type of conditionals. It's double when are combined with an `and` like this...
think it's safe like this